parts/k8s/containeraddons/kubernetesmasteraddons-azure-policy-deployment.yaml

```diff
@@ -0,0 +1,627 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Because the payload of this spec is so large, I'd expect it to push up against the ARM customData limits for cluster configurations that include other addons. Could you try building a cluster with a bunch of other addons enabled and verify that this is indeed the case? :(

Reply:
Deployed `azure-policy` with` cluster-autoscaler`, `aad-pod-identity` and `aci-connector` and deployment was successful. Do you think that's good enough, or I can test with a few more?

Reply:
Calico is another big one. You could try adding this to your existing "lots o' addons" config:

```
"kubernetesConfig": {
    ...
    "networkPolicy": "calico",
    "networkPlugin": "azure"
    ...
}
```

Reply:
Looks like adding calico hits the ARM length limit: `Custom data in OSProfile must be in Base64 encoding and with a maximum length of 87380 characters.` Any suggestions? Should I document this in the addon readme?

Reply:
Thanks for confirming.

Can you plan to file an issue after we merge this? I'd say this is just a "bug" in the platform overall, and should suggest to us that we should evolve the implementation away from delivering the entire k8s app layer spec (i.e., all these yaml files) via cloud-init.

Reply:
sgtm

Reply:
opened #2205